unikernel.ml

```diff
@@ -178,7 +187,8 @@ struct
         in
         Lwt.return
           (reply reqd ~content_type:"application/json"
-             (Utils.Status.to_json status))
+             (Utils.Status.to_json status)
+             `Bad_request)
```

Comment:
I guess esp. here where we have "status" (the json object carrying the reply code) and the polymorphic variant status we can refactor to deduplicate. this can be done in a separate PR.